stl/inc/istream

```diff
@@ -600,7 +600,7 @@ public:
     }
 
     _NODISCARD streamsize __CLR_OR_THIS_CALL gcount() const { // get count from last extraction
-        return _Chcount;
+        return _Chcount < 0 ? (numeric_limits<streamsize>::max)() : _Chcount;

```

Comment:
There are a few problems with this approach:
1. It doesn't address the root cause of the issue: integer overflow when counting `_Chcount`.
2. It is seemingly based on a __false assumption__ that signed integers wrap after overflow.
   Signed integer overflow is [undefined behavior](https://en.cppreference.com/w/cpp/language/ub). When a C++ program contains undefined behavior, the behavior of the _whole_ program becomes undefined.

Reply:
In this case it's necessary to make check _Chcount == (numeric_limits<streamsize>::max)() on every increment increment of _Chcount? Will it be appropriate?

Reply:
I believe it's necessary when overflowing is possible (some functions such as `istream::getline` could never overflow `_Chcount`).

Reply:
I've fixed it. Please watch if you can. Thanks